### DISCUSSION TITLE
issue with downloading exodus

### DISCUSSION THREAD

**Discussion by lacekim** [Homebrew/discussions#3140] [Casks]
```
➜  ~ brew install --cask exodus
/usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `require': cannot load such file -- sorbet-runtime-stub (LoadError)
	from /usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `require'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `<main>'
➜  ~
```

having a issue, not sure how to continue on. Yes I can go to the website and DL it that way but that will mean im no longer using this. Thanks a lot for helping and hope this gets addressed

**Comment by SMillerDev** [comment]
What do `brew doctor` and `brew config` say?

**Reply by lacekim** [reply]
```
➜  ~ brew config
HOMEBREW_VERSION: >=2.5.0 (shallow or no git repository)
ORIGIN: https://github.com/Homebrew/brew
HEAD: 33709297cf9cb37605cf4efcb7151905baf8ccf5
Last commit: 3 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 44e4f258908a39a0d9bd6c3c775df52c05acf004
Core tap last commit: 60 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: octa-core 64-bit kabylake
clang: error: write out empty cache file '/var/folders/h9/4j1dpsvd1x3cr96d3jv53x200000gn/T/xcrun_db' (errno=No space left on device)
2022-04-04 16:05:16.272 xcodebuild[4767:46049] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-04-04 16:05:16.273 xcodebuild[4767:46049] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
clang: error: write out empty cache file '/var/folders/h9/4j1dpsvd1x3cr96d3jv53x200000gn/T/xcrun_db' (errno=No space left on device)
2022-04-04 16:05:17.945 xcodebuild[4772:46080] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-04-04 16:05:17.945 xcodebuild[4772:46080] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
Clang: 13.1.6 build 1316
Git: 2.33.1 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.2.1-x86_64
CLT: 12.0.0.32.2
Xcode: 13.3
➜  ~
```

**Reply by lacekim** [reply]
I ran brew doctor before config

**Reply by SMillerDev** [reply]
Okay, and what did `brew doctor` say?

**Comment by lacekim** [comment]
➜  ~ brew install --cask exodus
Running `brew update --preinstall`...
error: unable to write file Library/Homebrew/Gemfile.lock
error: unable to write file Library/Homebrew/dev-cmd/pr-pull.rb
error: unable to write file Library/Homebrew/extend/os/mac/search.rb
error: unable to write file Library/Homebrew/sorbet/rbi/gems/json_schemer@0.2.20.rbi
error: unable to write file Library/Homebrew/test/dev-cmd/pr-pull_spec.rb
error: unable to write file Library/Homebrew/vendor/bundle/bundler/setup.rb
fatal: sha1 file '/usr/local/Homebrew/.git/index.lock' write error. Out of diskspace
error: unable to write file Library/Homebrew/Gemfile.lock
error: unable to write file Library/Homebrew/dev-cmd/pr-pull.rb
error: unable to write file Library/Homebrew/extend/os/mac/search.rb
error: unable to write file Library/Homebrew/sorbet/rbi/gems/json_schemer@0.2.19.rbi
error: unable to write file Library/Homebrew/test/dev-cmd/pr-pull_spec.rb
error: unable to write file Library/Homebrew/vendor/bundle/bundler/setup.rb
fatal: sha1 file '/usr/local/Homebrew/.git/index.lock' write error. Out of diskspace
error: failed to write new configuration file /usr/local/Homebrew/.git/config.lock
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.
/usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `require': cannot load such file -- sorbet-runtime-stub (LoadError)
	from /usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `require'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `<main>'
➜  ~

This is what im getting

**Reply by lacekim** [reply]
➜  ~ brew config
HOMEBREW_VERSION: >=2.5.0 (shallow or no git repository)
ORIGIN: https://github.com/Homebrew/brew
HEAD: 33709297cf9cb37605cf4efcb7151905baf8ccf5
Last commit: 3 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 44e4f258908a39a0d9bd6c3c775df52c05acf004
Core tap last commit: 60 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /usr/local
HOMEBREW_CASK_OPTS: []
HOMEBREW_CORE_GIT_REMOTE: https://github.com/Homebrew/homebrew-core
HOMEBREW_MAKE_JOBS: 8
Homebrew Ruby: 2.6.8 => /usr/local/Homebrew/Library/Homebrew/vendor/portable-ruby/2.6.8/bin/ruby
CPU: octa-core 64-bit kabylake
clang: error: write out empty cache file '/var/folders/h9/4j1dpsvd1x3cr96d3jv53x200000gn/T/xcrun_db' (errno=No space left on device)
2022-04-04 16:05:16.272 xcodebuild[4767:46049] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-04-04 16:05:16.273 xcodebuild[4767:46049] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
clang: error: write out empty cache file '/var/folders/h9/4j1dpsvd1x3cr96d3jv53x200000gn/T/xcrun_db' (errno=No space left on device)
2022-04-04 16:05:17.945 xcodebuild[4772:46080] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionSentinelHostApplications for extension Xcode.DebuggerFoundation.AppExtensionHosts.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
2022-04-04 16:05:17.945 xcodebuild[4772:46080] Requested but did not find extension point with identifier Xcode.IDEKit.ExtensionPointIdentifierToBundleIdentifier for extension Xcode.DebuggerFoundation.AppExtensionToBundleIdentifierMap.watchOS of plug-in com.apple.dt.IDEWatchSupportCore
Clang: 13.1.6 build 1316
Git: 2.33.1 => /usr/local/bin/git
Curl: 7.77.0 => /usr/bin/curl
macOS: 12.2.1-x86_64
CLT: 12.0.0.32.2
Xcode: 13.3
➜  ~ brew install --cask exodus
Running `brew update --preinstall`...
error: unable to write file Library/Homebrew/Gemfile.lock
error: unable to write file Library/Homebrew/dev-cmd/pr-pull.rb
error: unable to write file Library/Homebrew/extend/os/mac/search.rb
error: unable to write file Library/Homebrew/sorbet/rbi/gems/json_schemer@0.2.20.rbi
error: unable to write file Library/Homebrew/test/dev-cmd/pr-pull_spec.rb
error: unable to write file Library/Homebrew/vendor/bundle/bundler/setup.rb
fatal: sha1 file '/usr/local/Homebrew/.git/index.lock' write error. Out of diskspace
error: unable to write file Library/Homebrew/Gemfile.lock
error: unable to write file Library/Homebrew/dev-cmd/pr-pull.rb
error: unable to write file Library/Homebrew/extend/os/mac/search.rb
error: unable to write file Library/Homebrew/sorbet/rbi/gems/json_schemer@0.2.19.rbi
error: unable to write file Library/Homebrew/test/dev-cmd/pr-pull_spec.rb
error: unable to write file Library/Homebrew/vendor/bundle/bundler/setup.rb
fatal: sha1 file '/usr/local/Homebrew/.git/index.lock' write error. Out of diskspace
error: failed to write new configuration file /usr/local/Homebrew/.git/config.lock
error: cannot rebase: You have unstaged changes.
error: Please commit or stash them.
/usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `require': cannot load such file -- sorbet-runtime-stub (LoadError)
	from /usr/local/Homebrew/Library/Homebrew/standalone/sorbet.rb:11:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `require'
	from /usr/local/Homebrew/Library/Homebrew/startup/sorbet.rb:9:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/startup.rb:10:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/global.rb:4:in `<top (required)>'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `require_relative'
	from /usr/local/Homebrew/Library/Homebrew/brew.rb:29:in `<main>'
➜  ~

**Reply by gromgit** [reply]
>```
>fatal: sha1 file '/usr/local/Homebrew/.git/index.lock' write error. Out of diskspace
>```

And there's your problem. You need to clean up your disk.
